.github/workflows/checks.yml

```diff
@@ -77,7 +77,8 @@ jobs:
     # Dependabot PRs sadly also don't have access to secrets, so we skip them as well
     if:
       (github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository) &&
-      github.actor != 'dependabot[bot]'
+      github.actor != 'dependabot[bot]' &&
+      ${{ false }} # Debug ID backend changes broke the E2E tests, we need to revisit them
```

Comment:
not sure if this will work, nesting this inside of if here? I think the safer is `1 == 2` or something like this? Never 100% sure about this in the GHA yaml format 😅